modules.d/90multipath/multipath-shutdown.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for i in $(multipath -l -v1); do
+    if ! $(dmsetup table $i | sed -n '/.*queue_if_no_path.*/q1') ; then
```

Comment:
This returns on a machine with no multipath:
```
# multipath -l -v1
Jan 17 11:09:57 | /etc/multipath.conf does not exist, blacklisting all devices.
Jan 17 11:09:57 | A default multipath.conf file is located at
Jan 17 11:09:57 | /usr/share/doc/device-mapper-multipath-0.7.3/multipath.conf
Jan 17 11:09:57 | You can run /sbin/mpathconf to create or modify /etc/multipath.conf
Jan 17 11:09:57 | DM multipath kernel driver not loaded
```

Reply:
`$ multipath -l -v1`
is empty for me.

Seems what you are seeing is the effect of `0009-RH-don-t-start-without-a-config-file.patch` and `0011-RH-add-mpathconf.patch` in the Fedora RPMs.

It also seems that these messages go to stdout, rather than stderr, so simply ignoring stderr won't cut it. Can you check?

Reply:
Maybe prepend `[ -e /etc/multipath.conf  ] || exit 1` ?

Reply:
or better fix the fedora patches 😞

Reply:
Filed https://bugzilla.redhat.com/show_bug.cgi?id=1535868